src/terrain/util/config.clj

```diff
@@ -336,141 +336,140 @@
        (cfg/env-setting "DATA_INFO_PORT")
        (data-info-base-url)))))
 
-(cc/defprop-str tree-parser-url
+(cc/defprop-optstr tree-parser-url
   "The URL for the tree parser service."
   [props config-valid configs tree-viewer-routes-enabled]
-  "terrain.tree-viewer.base-url")
+  "terrain.tree-viewer.base-url" "http://portnoy.iplantcollaborative.org/parseTree")
 
-(cc/defprop-str es-url
+(cc/defprop-optstr es-url
   "The URL for Elastic Search"
   [props config-valid configs data-routes-enabled]
-  "terrain.infosquito.es-url")
+  "terrain.infosquito.es-url" "http://elasticsearch:9200")
 
-(cc/defprop-str jwt-private-signing-key
+(cc/defprop-optstr jwt-private-signing-key
   "The path to the private key used for signing JWT assertions."
   [props config-valid configs]
-  "terrain.jwt.signing-key.private")
+  "terrain.jwt.signing-key.private" "/path/to/private-key.pem")
```

Comment:
This doesn't seem like a useful default -- nobody's system is likely to have a `/path` directory.